spec/lib/user_content_report_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UserContentReport do
+  describe '#related_request_for_comment' do
```

Comment:
How about adding a subject here?
```suggestion
RSpec.describe UserContentReport do
  subject(report) { described_class.new(reported_content:) }

  describe '#related_request_for_comment' do
```

Reply:
Done.